### ISSUE TITLE
[Feature Request] Layout for vertical monitors?

### ISSUE THREAD

**Issue by adamhbz** [ForkIssues/Tracker#1937]
Hi,

Just wondering if it is possible to customise the layout. When fork is opened on a vertical monitor you are often left with a lot of vertical space and not much horizontal space.

- Would it be possible to collapse the 'branch' menu?
- Would it be possible to show the 'change preview' and 'commit message' on the lower half of the screen leaving the top have to view the files?

Example:
![Screenshot 2023-07-19 at 14 52 55](https://github.com/fork-dev/Tracker/assets/13640398/149b218e-80e1-4432-8f17-949591fe4635)

**Comment by DanPristupov** [comment]
No, it won't be possible. The layout customization is difficult and we don't have a time for that.

**Comment by HealsCodes** [comment]
OP has a valid point here though - currently Fork wastes a lot of Vertical *AND* horizontal space even on small screens and has no way to alleviate it. 

Maybe an option to have the commit history side-by-side with the commit details, without any other customisation (I know offering UI customisation is a lot of work) would be an enhancement.

 It would greatly reduce the wasted screen-space horizontally and allow those of us with vertical monitors to see a lot more of the history without having to sacrifice the height of the commit details.

For comparison - Fork's horizontal-favouring layout:
![Screenshot 2024-02-28 at 20 29 05@2x](https://github.com/fork-dev/Tracker/assets/304352/d329e874-24ec-4a95-8b1d-68c04e722c6e)

&lt;random other client with a vertically layout option enabled (changed files are shown inline the same way Fork does in Commit mode) &gt;
![Screenshot 2024-02-28 at 20 30 42@2x](https://github.com/fork-dev/Tracker/assets/304352/18d5731a-a6cb-4c68-8bc3-88aebd34460c)

**Comment by DanPristupov** [comment]
@HealsCodes there is an issue for vertical layout (https://github.com/fork-dev/Tracker/issues/756)
